ugali/analysis/loglike.py

```diff
@@ -309,6 +309,44 @@ def calc_signal_color2(self, distance_modulus, mass_steps=1000):
     # FIXME: Need to parallelize CMD and MMD formulation
     calc_signal_color = calc_signal_color1
 
+    def calc_surface_intensity(self, factor=10):
```

Comment:
@apace7 this is the new `calc_surface_intensity` function that does the adaptive gridding for evaluating the model predicted counts.